cpanfile

```diff
@@ -1,4 +1,5 @@
 requires 'HTML::Entities';
+requires 'HTML::Lint::Pluggable';
```

Comment:
`HTML::Lint::Pluggable` depends on `HTML::Lint` and this is a quote from the latest release notes of HTML::Lint:

> Note that this very well may be the final release of HTML::Lint that I make. I've been spending my time on HTML::Tidy5, which works on HTML5, checks for more problems, and is much faster.

Should we consider HTML::Tidy5 instead?

Reply:
I did, but it relies on an [external dependency](https://metacpan.org/pod/HTML::Tidy5#INSTALLING-TIDY).

Since the `HTML::Lint::Pluggable` still works, and apparently can catch some real problems, I decided to go for it. It also only added an extra second or so to lint all pages, so I am fine with that speed.

Reply:
ok. lets accept that then.